### ISSUE TITLE
Correct loss in GAN implementation?

### ISSUE THREAD

**Issue by zjost** [timsainb/tensorflow2-generative-models#8]
In [this notebook](https://github.com/timsainb/tensorflow2-generative-models/blob/master/2.0-GAN-fashion-mnist.ipynb) I noticed the following line in the `GAN.compute_loss` method:  `gen_loss = gan_loss(logits=logits_x_gen, is_real=True)`.  Shouldn't `is_real=False`?

**Comment by zjost** [comment]
Closing, as I'm seeing now this is the intended implementation, which is a heuristic objective meant to provide stronger gradients to the Generator.
